tests/test_app.py

```diff
@@ -1,4 +1,228 @@
-import todo_app
+# from todo_app.data.trello_items import Item # <<< DONT KNOW WHY I CANT IMPORT THIS WITHOUT CAUSING TEST DISCOVERY ISSUES
```

Comment:
I agree we want to use this version of the item class - I don't see discovery issues myself but one _possible_ reason is because VS code is noticing that the way you use `Item` isn't consistent with this class definition - your example `Item` below is missing the `id` field in the constructor (`__init__` method) so when you create individual items you're not passing that through

Switching this import back on, removing the Item class below, and adjusting all the object creations to pass through the `id` field allowed your tests to go back to passing through the VSCode interface for me